scripts/gulp/plugins/gulp-component-menu-behaviors.ts

```diff
@@ -64,7 +63,23 @@ export default () => {
       if (!variation.specification) {
         const variationName = variation.name.replace('.ts', '');
         const definitionName = `${variationName}Definition`;
-        const definition = behaviorDefinitions[definitionName];
+
+        const absPathToBehaviorDefinition = path.normalize(
+          path.join(
+            require.resolve(`@fluentui/a11y-testing`),
+            '../',
+            'definitions',
+            behaviorName,
+            `${definitionName}.ts`,
+          ),
```

Comment:
nit: should we use `path.allPackages()`?

```suggestion
          paths.allPackages(
            'a11y-testing',
            'src',
            'definitions',
            behaviorName,
            `${definitionName}.ts`,
          ),
```